modules/tinymce/CHANGELOG.md

```diff
@@ -84,6 +84,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Replaced 'Powered by Tiny' link text with logo #TINY-8371
 - The `mceInsertTable` command can no longer open the insert table dialog. Use the `mceInsertTableDialog` command instead #TINY-8273
 - Moved non-UI table functionality to core #TINY-8273
+- The `table_style_by_css` option now defaults to `true` #TINY-8259
+- The `table_use_colgroups` option now defaults to `true` #TINY-8259
```

Comment:
```suggestion
- The `table_style_by_css` option has been set to `true` by default #TINY-8259
- The `table_use_colgroups` option has been set to `true` by default #TINY-8259
```
Maybe to match with other similar changes above? Either that or change the ones above to match this format, as tbh I do prefer what you have.

Reply:
I have made it the same as the other entries